README.md

```diff
@@ -0,0 +1,27 @@
+# Combirepo
+
+Combirepo is a tool for combining several repositories into firmware.
+
+Intended to be used together with OBS-produced repositories.
+Refer man 1 combirepo for details.
+
+## Installation
+
+Use either docker build:
+```
+$ ./setup.py sdist
+$ docker build -t combirepo ./
+$ docker run -Pti combirepo /bin/bash
+```
+
+Or `bdist_deb`:
+```
+$ python -m pip install stdeb --user
+$ ./setup.py sdist
```

Comment:
Seems that `setup.py` isn't executable. Don't we usually call it as `python setup.py` instead of `./setup.py`?

Reply:
It *is* an executable, though:
```
$ ll ./setup.py
-rwxrwxr-x 1 dnikiforov dnikiforov 3.5K Jun 20 17:58 ./setup.py
```
Either way is fine.

Reply:
But maybe shebang should be changed to `#!/usr/bin/python2`.

Reply:
OK

Reply:
Also, please mention requirements somewher, i.e install yum, iniparse and so on.

Reply:
Yes, good point. They're listed in `setup.py`:
```
      install_requires=['iniparse',
                        'python-igraph',
                        'configparser'],
```
So we can write it both in `Readme.md` and in `setup.py`.

Reply:
And also here: `requires=['yum', 'mic', 'createrepo', 'modifyrepo']`.

Reply:
I thought that our `setup.py` will resolve dependencies itself, won't it?

Reply:
Well, it doesn't done this for me. Also error may be caused by i.e. obsolete pip, or angry firewall settings, so it is better to mention it somewhere, I think.

Reply:
OK, then, adding such list of requirements to the REAME sounds quite appropriate.

Reply:
ping?

Reply:
Sorry, @p-privalov is on a vacation currently.